src/compiler/checker.ts

```diff
@@ -34353,6 +34353,12 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
     function checkExpressionWithTypeArguments(node: ExpressionWithTypeArguments | TypeQueryNode) {
         checkGrammarExpressionWithTypeArguments(node);
         forEach(node.typeArguments, checkSourceElement);
+        if (node.kind === SyntaxKind.ExpressionWithTypeArguments) {

```

Comment:
I probably want to check this differently, because I think I'm missing this:

```ts
maybeBox instanceof import("./box").Box<number>
```

Reply:
Hm, no, that'd be broken for other reasons. I was looking for something that would be a `TypeQueryNode` but I don't think that can even appear as a child of an `instanceof`.